js/game.js

```diff
@@ -5,8 +5,75 @@ export default class Game {
     this.tamagotchi = new Tamagotchi();
   }
 
-  start = ({ healthElement, hungerElement, energyElement, funElement }) => {
-    this.tamagotchi.mount({ healthElement });
+  start = ({
+    healthElement,
+    hungerElement,
+    energyElement,
+    funElement
+  }) => {
+    this.tamagotchi.mount({
+      healthElement,
+      hungerElement,
+      energyElement,
+      funElement,
+    });
+
     console.log("Game started");
+
+    const decreaseHealth = () => {
+      if (this.tamagotchi.hunger.value <= 0) this.tamagotchi.health.value--;
+      this.tamagotchi.mount({
+        healthElement,
+        hungerElement,
+        energyElement,
+        funElement,
+      })
+      
+      if (this.tamagotchi.health.value <= 0) {
```

Comment:
**nitpick:** We should extract this (interval clears) as a separate function, and call it for example `endGame`. You should have a possibility to run it whenever you want without being forced to duplicate this logic